examples/sine.js

```diff
@@ -5,8 +5,8 @@
  * https://web.archive.org/web/20110816002016/http://blogs.msdn.com/b/dawate/archive/2009/06/24/intro-to-audio-programming-part-3-synthesizing-simple-wave-audio-using-c.aspx
  */
 
-const Readable = require('stream').Readable
-const bufferAlloc = require('buffer-alloc')
+const { Readable } = require('stream')
+const { Buffer } = require('buffer')
```

Comment:
`Buffer` is a global in Node.js, I haven't seen much code in the wild that imports it? Is there any upside to that 🤔 

```suggestion
```

Reply:
I know...

ever since xo started [doing it](https://github.com/xojs/xo/commit/9f400745a1404d0098a853f3b33508e0546f16c8) i thought "ok, why not do it?" what could possible be the reason for it?
 
- If you don't require/import Buffer then you have the rule that `no-undefined` or "no global" will throw an error for in many cases/projectsd.
- it makes it harder for other compiles to "walk" the AST and figure out if something is using some global stuff and then polyfill it.
  - it just makes something as simple as Ryan dhal stated: just include that little extra extension for imports, it makes it much easier for the resolver to figure out what to include  
- You also got the Blob at the buffer module `import { Blob, } from 'buffer'` so why not use `require('buffer').Buffer for the same reason?`
  - make it explicitly and you will know what you depend upon
  - i don't know if it can help make some stuff friendlier for Deno also
- another reason could possible be to use sourcemaps to override what is imported